.github/CHANGELOG.md

```diff
@@ -175,7 +177,8 @@
 
 This release contains contributions from (in alphabetical order):
 
-Aroosa Ijaz, Juan Miguel Arrazola, Thomas Bromley, Jack Ceroni, Josh Izaac, Antal Száva
+Aroosa Ijaz, Juan Miguel Arrazola, Thomas Bromley, Jack Ceroni, Josh Izaac, 
+Nathan Killoran, Robert Lang, Antal Száva
```

Comment:
@ral9000 please add any others who contributed to your PR (even via detailed code review)

Reply:
Done, please see the Contributions section in my suggested changelog description.